src/Microsoft.Health.Dicom.Core/Features/Workitem/WorkitemSerializer.cs

```diff
@@ -27,13 +27,14 @@ public async Task<IEnumerable<DicomDataset>> DeserializeAsync(Stream stream, str
             EnsureArg.IsNotNull(stream, nameof(stream));
             EnsureArg.IsNotEmptyOrWhiteSpace(contentType, nameof(contentType));
 
-            if (!string.Equals(contentType, KnownContentTypes.ApplicationJson, StringComparison.OrdinalIgnoreCase))
+            if (!string.Equals(contentType, KnownContentTypes.ApplicationJson, StringComparison.OrdinalIgnoreCase) &&
```

Comment:
Do we need to still support ApplicationJson

Reply:
No, it's not needed.